rules/no-zero-fractions.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const message = 'Zero fraction or dangling dot in number.';
```

Comment:
Maybe the message can be improve by suggesting how to fix the problem ?

Reply:
If you have some particular message in mind, please suggest. Otherwise, I think it's pretty self describing, plus it's a fixable error, so expected to be fixed automatically for codebases with fixing set up.

Reply:
Messages usually don't contain the solution if they're auto-fixable. I do think the message could be more actionable. How about:

> Don't use a zero fraction or dangling dot in the number.

Reply:
This made me think. We should probably have separate error messages for these two cases to make it clearer at least:

> Don't use a zero fraction in the number.

> Don't use a dangling dot in the number.